src/main/java/com/o19s/es/ltr/feature/store/StoredFeatureSet.java

```diff
@@ -175,6 +175,36 @@ public StoredFeatureSet append(List<StoredFeature> features) {
         return new StoredFeatureSet(name, newFeatures);
     }
 
+    /**
+     * Generate a new set by merging features with this set, feature with same names are updated new features are appended.
```

Comment:
Hmm I thought we issued errors when we tried to append a feature to a featureset when a name was already taken.

Reply:
This is on purpose, you'll have the choice to merge the features.
By default it'll be like you said, features are always appended and will fail if a name already exists.
But adding `?merge=true` to the request url will instruct the plugin to merge and overwrite features.

Reply:
Oh I see. Understood